code/modules/clothing/under/miscellaneous.dm

```diff
@@ -388,6 +388,9 @@
 	name = "flower dress"
 	desc = "A beautiful dress with a skirt of flowers."
 	icon_state = "flower_dress"
+	sprite_sheets = list(

```

Comment:
Is this not inherited from the default under clothing?

Reply:
I have no clue, is it? Cerebul gave me the code, I'm a clueless babbit.

Reply:
You're right it's applied to all clothing/under items already, I assumed it wasn't. You can take this bit of code out schnee

Reply:
Will do! Let me remember how to do that and I'll update. o7